Source/MoreDotNet.Test/MoreDotNet.Tests.csproj

```diff
@@ -40,9 +40,13 @@
     <CodeAnalysisRuleSet>MoreDotNet.Tests.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <Private>False</Private>
+    </Reference>
```

Comment:
Please remove the reference to Microsoft.VisualStudio.QualityTools.UnitTestFramework.